tests/components/modbus/conftest.py

```diff
@@ -0,0 +1,96 @@
+"""The tests for the Modbus sensor component."""
+from datetime import timedelta
+import logging
+from unittest import mock
+
+import pytest
+
+from homeassistant.components.modbus.const import (
+    CALL_TYPE_REGISTER_INPUT,
+    CONF_REGISTER,
+    CONF_REGISTER_TYPE,
+    CONF_REGISTERS,
+    DEFAULT_HUB,
+    MODBUS_DOMAIN,
+)
+from homeassistant.const import CONF_NAME, CONF_PLATFORM, CONF_SCAN_INTERVAL
+from homeassistant.setup import async_setup_component
+import homeassistant.util.dt as dt_util
+
+from tests.common import MockModule, async_fire_time_changed, mock_integration
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def mock_hub(hass):
+    """Mock hub."""
+    mock_integration(hass, MockModule(MODBUS_DOMAIN))
+    hub = mock.MagicMock()
+    hub.name = "hub"
+    hass.data[MODBUS_DOMAIN] = {DEFAULT_HUB: hub}
+    return hub
+
+
+common_register_config = {CONF_NAME: "test-config", CONF_REGISTER: 1234}
```

Comment:
Constants should be all caps, ie `COMMON_REGISTER_CONFIG`.

Reply:
That is correct, an even better solution is to remove the variable since it is unused. This is done in a PR I am preparing.